selenium-webdriver/firefox/index.js

```diff
@@ -515,6 +515,63 @@ class ServiceBuilder extends remote.DriverService.Builder {
   enableVerboseLogging(opt_trace) {
     return this.addArguments(opt_trace ? '-vv' : '-v');
   }
+
+  // TODO:
```

Comment:
Maybe we should remove this section of commented out code until it's ready to land?

Reply:
Done.